pulp_container/app/models.py

```diff
@@ -239,7 +326,9 @@ def finalize_new_version(self, new_version):
         validate_repo_version(new_version)
 
 
-class ContainerPushRepository(Repository, AutoAddObjPermsMixin, AutoDeleteObjPermsMixin):
+class ContainerPushRepository(
+    Repository, CreateGroupAndAddPermissionsMixin, AutoDeleteObjPermsMixin
```

Comment:
We are not deleting groups here because all groups are related to the ContainerDistribution, is my understanding correct?